app/tests/test_queries.py

```diff
@@ -1,210 +1,281 @@
-                "test_file5.txt",
-                files[4].consignment.series.body.BodyId,
-                files[4].consignment.series.SeriesId,
-            ),
+                file.consignment.series.body.Name,
+                file.consignment.series.Name,
+                file.consignment.ConsignmentReference,
+                file.FileName,
+                file.consignment.series.body.BodyId,
+                file.consignment.series.SeriesId,
+            )
         ]
-        assert result.pages > 0
-        assert result.has_next is True
 
-    def test_fuzzy_search_get_specific_page_results(self, client: FlaskClient):
+    def test_build_fuzzy_search_query_no_results(self, client: FlaskClient):
         """
-        Given multiple File objects in the database
-            and a query string that matches some fields in only 1 of them
-        When fuzzy_search is called with the query
-        Then a list containing 5 (per_page) dictionary with information for the corresponding
-            file is returned for page 2
+        Given a query string that does not match any field in any file object
+            in the database
+        When fuzzy_search is called with it and is executed
+        Then an empty list is returned
         """
-        files = create_multiple_test_records()
+        FileFactory(FileName="foo", FileType="file")
+        query_string = "bar"
+        query = build_fuzzy_search_query(query_string)
+        results = query.all()
+        assert results == []
 
-        query = "testing body"
-        result = fuzzy_search(query, page=2, per_page=per_page)
-        assert result.items == [
-            (
-                "testing body6",
-                "test series6",
-                "test consignment6",
-                "test_file6.txt",
-                files[5].consignment.series.body.BodyId,
-                files[5].consignment.series.SeriesId,
-            ),
-            (
-                "testing body7",
-                "test series7",
-                "test consignment7",
-                "test_file7.txt",
-                files[6].consignment.series.body.BodyId,
-                files[6].consignment.series.SeriesId,
-            ),
-            (
-                "testing body8",
-                "test series8",
-                "test consignment8",
-                "test_file8.txt",
-                files[7].consignment.series.body.BodyId,
-                files[7].consignment.series.SeriesId,
-            ),
-            (
-                "testing body9",
-                "test series9",
-                "test consignment9",
-                "test_file9.txt",
-                files[8].consignment.series.body.BodyId,
-                files[8].consignment.series.SeriesId,
-            ),
-        ]
-        assert result.pages > 0
-        assert result.has_next is False
-        assert result.has_prev is True
+
+#     # def test_build_fuzzy_search_query_multiple_matches(self):
```

Comment:
Possible to remove these is they're not needed?

Reply:
ah yeah thanks for spotting these, I had put them as placeholders. will remove for now to get unblocked, can add more later